### PR TITLE
refactor server.js to routes/authentication and routes/main

### PR DESCRIPTION
**Pull request by dave2000sang**

### CODE REVIEW

File: routes/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const path = require('path')
+
+// verify token and render app.html
+router.get('/app', (req, res) => {
+	console.log("app route received")
+	res.sendFile(path.join(__dirname, '/../','main', 'public', 'index.html'))
+})
+
+
+module.exports = router
```
Comment (1298se) [inline routes/main.js]: missing End of file new line

File: routes/authentication.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		res.status(400).send('Bad Request: ID Token is null')
+	} else {
+		admin.auth().verifyIdToken(token).then(function (decodedToken) {
+			res.sendStatus(200)
+		}).catch(function (error) {
+			console.log("error logging in user")
+		})
+	}
+})
+router.post('/register', (req, res) => {
```
Comment (1298se) [inline routes/authentication.js]: add space between methods

File: routes/authentication.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+const express = require('express')
+var router = express.Router()
```
Comment (1298se) [inline routes/authentication.js]: use const for constants

File: routes/main.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+var router = express.Router()
+const path = require('path')
+
+// verify token and render app.html
```
Comment (1298se) [inline routes/main.js]: use const for constants

File: server.js [hunk 5]
```diff
@@ -1,22 +1,14 @@
 // server.js is the backend script for the server
 const express = require('express')
 const bodyParser = require('body-parser')
-const admin = require('firebase-admin')
 const path = require('path')
+var authentication = require('./routes/authentication')
```
Comment (1298se) [inline server.js]: const for constants

File: routes/authentication.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+const admin = require('firebase-admin')
+
+const serviceAccount = require("../ServiceAccountKey")
+admin.initializeApp({
+	credential: admin.credential.cert(serviceAccount)
+})
+let db = admin.firestore()
```
Comment (1298se) [inline routes/authentication.js]: const db

File: routes/main.js (hunk 1 above)
Comment (virajg224) [reply routes/main.js]: we should set up a linter or something to detect these

**Comment by notoriousmango** [comment]
This is closed because we are using typescript and i already refactored it

**Pull request closed without merging**